test/core/extensions/coreIssuanceOrder.spec.ts

```diff
@@ -93,16 +94,21 @@ contract("CoreIssuanceOrder", (accounts) => {
     let components: StandardTokenMockContract[] = [];
     let componentUnits: BigNumber[];
     let setToken: SetTokenContract;
```

Comment:
Let's tighten these up, remove any variables here, also in `cancelOrder` for variables that don't need to be toggled. 

`naturalUnit`, `components`, `componentUnits`, `setToken`